### ISSUE TITLE
[Tech Request]: Add account id to object path

### ISSUE THREAD

**Issue by LeftHandCold** [matrixorigin/matrixone#9376]
### Is there an existing issue for the same feature request?

- [X] I have checked the existing issues.

### Describe the feature you'd like

Add account id to object path to facilitate mo expansion.

**Comment by XuPeng-SH** [comment]
A object file can be shared by multiple table (may be different account) and I don't think it is a valid requirement now.

**Comment by XuPeng-SH** [comment]
i don't think it is a valid request. since an object will be shared by different account
